lib/std/crypto/25519/edwards25519.zig

```diff
@@ -221,7 +221,7 @@ pub const Edwards25519 = struct {
         pc[1] = p;
         var i: usize = 2;
         while (i <= count) : (i += 1) {
-            pc[i] = pc[i - 1].add(p);
+            pc[i] = if (i % 2 == 0) pc[i / 2].dbl() else pc[i - 1].add(p);
```

Comment:
Just wondering, does Zig have an equivalent of [Python's `divmod`](https://docs.python.org/3/library/functions.html#divmod)?
This way in the case of `== 0` you would already have the result in a variable.

Reply:
The compiler gets that `I%2==0` will be true every other iteration. In `ReleaseSmall` mode, both variants are executed in a row, in `ReleaseFast` mode, everything is inlined.